tests/bud_overlay_leaks.bats

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "bud overlay storage leaked mount" {
+  if test \! -e /usr/bin/fuse-overlayfs -a "$BUILDAH_ISOLATION" = "rootless"; then
+    skip "BUILDAH_ISOLATION = $BUILDAH_ISOLATION" and no /usr/bin/fuse-overlayfs present
+  fi
+
+  target=pull
+  run_buildah 125 --storage-driver=overlay bud --signature-policy ${TESTSDIR}/policy.json -t ${target} --pull-never ${TESTSDIR}/bud/pull
+  expect_output --substring "no such image"
+
+  leftover=$(mount | grep $TESTDIR)
```

Comment:
I edited my comment, but was too late: this is going to fail, because the `grep` will fail (under normal circumstances), which will cause BATS to exit. Could you add `| cat` after the grep?
```bash
   leftover=$(mount | grep $TESTDIR | cat)
```
Sorry for not getting it right the first time.

Reply:
done